src/Modules/OrchardCore.Commerce/Views/ShoppingCartCell_OldPrice.cshtml

```diff
@@ -1,8 +1,9 @@
-@using OrchardCore.Commerce.Models;
-@using OrchardCore.Commerce.Tax.Extensions
 @using OrchardCore.Commerce.Abstractions.ViewModels
-
-@inject OrchardCore.Settings.ISiteService SiteService
+@using OrchardCore.Commerce.Models
+@using OrchardCore.Commerce.Promotion.Extensions
+@using OrchardCore.Commerce.Tax.Extensions
+@using OrchardCore.Settings
+@inject ISiteService SiteService
```

Comment:
is this used anywhere?

Reply:
Yes, [on line 11](https://github.com/OrchardCMS/OrchardCore.Commerce/blob/c8a988c4a534f479a188262bf30b32a256a31101/src/Modules/OrchardCore.Commerce/Views/ShoppingCartCell_OldPrice.cshtml#L11).

Reply:
I looked but did not see it. Uh


That line can also be simplified using the new extension 

await SiteService.GetSettingsAsync<PriceDisplaySettings>();